layers/ubuntu1604/bazel/extract_version.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is autogenerated by the dependency update service and should not be modified directly.
+# For more details, check the deps_spec.yaml file in the current folder.
+
+# This script accepts a tarball with the bazel binary available at
+# "usr/local/lib/bazel/bin" and produces a YAML file with the bazel version
+# which looks as follows:
+# packages:
+# - name: bazel
+#   version: "<version string>"
+# where <version string> is the Bazel version, e.g., 0.27.1
+
+set -e
+
+input_tar=$1
+output_yaml=$2
+
+echo "Input tar: " $input_tar
+echo "Output YAML: " $output_yaml
+
+echo "Extracting the tarball into the current directory..."
+tar xf $input_tar -C $PWD
+echo "Running Bazel and extracting the version string"
+# Runs Bazel specifying a directory in the runfiles directory as the bazel
```

Comment:
Runs the Bazel version we just extracted to a relative directory specifying ...

Reply:
Done